packages/core/src/render3/instructions/control_flow.ts

```diff
@@ -43,6 +43,67 @@ import {
   removeLViewFromLContainer,
 } from '../view/container';
 
+export function ɵɵconditionalCreate(
+  index: number,
+  templateFn: ComponentTemplate<any> | null,
+  decls: number,
+  vars: number,
+  tagName?: string | null,
+  attrsIndex?: number | null,
+  localRefsIndex?: number | null,
+  localRefExtractor?: LocalRefExtractor,
+): typeof ɵɵconditionalCreateBranch {
+  performanceMarkFeature('NgControlFlow');
+  const lView = getLView();
+  const tView = getTView();
+  const attrs = getConstant<TAttributes>(tView.consts, attrsIndex);
+  declareTemplate(
+    lView,
+    tView,
+    index,
+    templateFn,
+    decls,
+    vars,
+    tagName,
+    attrs,
+    localRefsIndex,
+    localRefExtractor,
+    [TNodeFlags.isControlFlowStart, TNodeFlags.isInControlFlow],
```

Comment:
nit: we can avoid creating an array here and just combine the flags here (similarly for the other instruction):
```suggestion
    TNodeFlags.isControlFlowStart | TNodeFlags.isInControlFlow,
```